README.md

```diff
@@ -68,6 +68,8 @@ GitHub Template Repository for ScribeMD
         since we use a
         [rebase-based workflow](https://git-scm.com/book/en/v2/Git-Branching-Rebasing).
   - [ ] [Configure automatic deletion of head branches when pull requests are merged](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches).
+  - [ ] [Configure automatic version bumps through Commitizen](https://commitizen-tools.github.io/commitizen/bump/)
+        by setting `bump: true` in the `Run Pre-commit Hooks` test in [`.github/workflows/test.yaml`](.github/workflows/test.yaml).
```

Comment:
"setting `bump: true`" --> "remove `bump: false`" since it's the default argument. Remove "in the `Run Pre-commit Hooks` test" for brevity as the file is very short, and it can be tricky to remember to update this if we were to change the name of the test.